index.js

```diff
@@ -0,0 +1,43 @@
+// const qs = require("qs")
+
+document.addEventListener("DOMContentLoad", () => {
+    console.log("hello")
+    // let welcomeBtn = document.querySelector("#welcomeBtn")
+
+
+
+    let classBtn = document.querySelector("#classBtn");
+    let studentBtn = document.querySelector("#studentBtn");
+    let fsBtn = document.querySelector("fsBtn");
+    let classDisplay = document.querySelector("#newClassDisplay")
+
+
+    let url = "http://localhost:3000/school"
+
+    const displayNewClass = (url) => {
+        
+    }
+
+
+
+
+
+
+    // addClassButton
+
+
+classBtn.addEventListener("submit", async (e) => {
```

Comment:
You should set your event listened to your form, not the button. That way if the person presses enter, it'll still go through. Button event listener would be listening to the "click".
That's why your forms still refresh the page on submit.